addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -84,7 +84,8 @@ export class Chatbot extends Record {
         return (
             (this.currentStep?.isLast &&
                 (!this.currentStep.expectAnswer || this.currentStep?.completed)) ||
```

Comment:
is this line still needed? Same for isLast?

Reply:
Yes it's sadly, otherwise we can't restart the chatbot.